.gitignore

```diff
@@ -38,4 +37,10 @@ gradle-app.setting
 !gradle-wrapper.jar
 
 # Cache of project
-.gradletasknamecache
\ No newline at end of file
+.gradletasknamecache
+
+# PerformanceTest results #
+Results
```

Comment:
Can you specify the exact path here? "\PerformanceTestCMDApplication\PerformanceTest_Result\Results", right?